workloads/kube-burner/README.md

```diff
@@ -93,13 +100,36 @@ It creates n-replicas of an application deployment (hello-openshift) and a servi
 It creates as many "sleep" pods as configured in the environment variable `PODS`.
 
 
-### Remote configuration
+### Launching custom workloads
 
-Apart from the pre-defined workloads and metric profiles available in benchmark-operator, you can create a benchmark that uses a remote configuration, metric or alert profile. These files must be accesible through HTTP protocol by the kube-burner job. The following environment variables can be used to configure the source for the different configuration files:
+Apart from the pre-defined workloads and metric profiles available in this repo, you can use your own benchmark, metric-profile and alert-profile by using the remote configuration feature of kube-burner. This feature allows kube-burner to fetch configuration files from remote locations using the HTTP protocol. These files must be accesible through HTTP protocol by the kube-burner job. The following environment variables can be used to configure the source for the different configuration files:
```

Comment:
> This feature allows kube-burner to fetch configuration files from remote locations using the HTTP protocol. These files must be accesible through HTTP protocol by the kube-burner job.

A little repetitive about the HTTP protocol, maybe:

> This feature allows kube-burner to fetch configuration files from remote locations via HTTP. 

Just a thought. Otherwise just `accesible` -> `accessible`.